src/reducers/editorialWorkflow.js

```diff
@@ -76,6 +77,15 @@ const unpublishedEntries = (state = null, action) => {
       // Update Optimistically
       return state.deleteIn(['entities', `${ action.payload.collection }.${ action.payload.slug }`]);
 
+    case UNPUBLISHED_ENTRY_REGISTER_DEPENDENCY:
+      return state.setIn([
```

Comment:
This is currently storing the dependencies in `entry.data.dependencies`, where it is then grabbed in `persistUnpublishedEntry`, added to `options`, and stored in metadata in `editorialWorkflowGit`. Should this be saving directly to `entry.metaData` instead?